config/routes.rb

```diff
@@ -22,4 +22,9 @@
   delete 'comments/:id', to: 'comments#destroy'
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get '/api/users/:user_id/posts', to: 'api/posts#index'
+  get '/api/comments/:post_id', to: 'api/comments#index'
```

Comment:
- [ ] Please I suggest that the GET  API endpoint that returns all comments for a post should be `/users/:user_id/posts/:id/comments` which needs both the user `id` and the post `id` to ensure the comment will be returned is for the user who `id` is taken.